tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
```

Comment:
`tests/View/Mix/stubs/hmr/hot` and `tests/View/Mix/stubs/hmr/hot-custom` don't have any other purpose than to exist.
I believe they should not be committed.

Instead you can touch these 2 files in `setUpBeforeClass` and remove them in `tearDownAfterClass`